catalog/app/containers/Bucket/CodeSamples/Code.tsx

```diff
@@ -52,7 +52,11 @@ function LineOfCode({ lang, text, help }: LineOfCodeProps) {
   const classes = useLineOfCodeStyles()
   return (
     <div className={classes.root}>
-      {highlight(text, lang)}
+      {lang === 'uri' ? (
+        <StyledLink href={text}>{text}</StyledLink>
```

Comment:
doesn't that need a target=blank or smth?

Reply:
It's a URL that a browser can't open. So, with target="_blank" it will open literally blank tab :)
If opening URL on the same page, browser handles URL, asking on how to open that URL, but the page will be preserved.

Reply:
Though, thinking in general, I can add target="blank" when URL starst from 'http'